tensorbay/client/dataset.py

```diff
@@ -294,7 +294,7 @@ def _generate_segments(
         return response["totalCount"]  # type: ignore[no-any-return]
 
     def _generate_segment_diffs(
-        self, basehead: str, offset: int = 0, limit: int = 128
+        self, basehead: str, offset: int = 0, limit: int = 10
```

Comment:
commit message：
`fix(client): fix the typo of "params" and adjust the limit in diff api`